org.genivi.commonapi.wamp.cli.product/commonapi_wamp_cli.product

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?pde version="3.5"?>
+
+<product name="CommonAPI WAMP Code Generator" uid="org.genivi.commonapi.wamp.cli.product" id="org.genivi.commonapi.wamp.cli.product" application="org.genivi.commonapi.console.application" version="0.7.0.qualifier" useFeatures="true" includeLaunchers="true">
+
+   <configIni use="default">
+   </configIni>
+
+   <launcherArgs>
+      <programArgs>-sp &quot;/Users/aadavani/All gits 11.09.2018/git 04.03.2019/capicxx-wamp-tools/org.genivi.commonapi.wamp.cli/test/&quot; -d &quot;/Users/aadavani/All gits 11.09.2018/git 04.03.2019/capicxx-wamp-tools/org.genivi.commonapi.wamp.cli/test/op&quot;
```

Comment:
Are the absolute paths "/Users/aadavani" in the program args example settings? Or could they be removed?

Reply:
Hello Klaus,
These are the program arguments that are required for the successful execution of the command line generator.

Of course they can be removed and the command line generator will complain about correct usage of it. I missed to remove them.